setup_environment_test.go

```diff
@@ -329,6 +331,19 @@ func organizationIsAdded(table *gherkin.DataTable) (err error) {
 
 	organization := getTableValue(table, "organization")
 
+	return NewCommand().
+		Run("snet organization create %s --registry-at %s -y", organization, registryAddress).
+		Err()
+}
+
+func organizationIsAdded2(table *gherkin.DataTable) (err error) {
```

Comment:
Are this function and organizationIsAdded used?

Reply:
I removed organizationIsAdded2 function.
organizationIsAdded functions is still used to add an organization.